applications/Anchor.md

```diff
@@ -0,0 +1,286 @@
+# Anchor, On-chain Linked List pallet and Name Service
+
+- **Team Name:** Fuu
+- **Payment Address:** 0xF8105ea1cC3130501bad5De60Bd3D794a9115dE2 (USDT)
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 1
+
+
+## Project Overview :page_facing_up:
+
+### Overview
+
+- Anchor is a on-chain linked list system base on substrate 3.0.0. It is used to support cApp ( Chain Application ) development. On another hand, Anchor can alse be treated as Name Service or on-chain key-value storage system.
+
+- Anchor is a isolated substrate pallet. It is currently used in the application of freeSaying. It can provide flexible data structure on the chain and handle complex logic without upgrading the substrate node itself.
+You can access the [freeSaying ( Only applicable to mobile )](https://freesaying.net) demo to know well. Anchor network is the very important basic storage system. Anchor.js is a isolate JS library to access Anchor network, can read and write data easily.
+Anchor pallet is a part of EasyPolka, the relationship as follow :
+
+![easypolka.png](http://android.im/anchor/easypolka.png)
+
+QR to access to freesaying.net.
+![easypolka.png](http://android.im/anchor/qr.png)
+
+- With Anchor, you can use run a substate network just need a bit upgrade. Through the highly customizable data structure and the ability of cApp, the threshold of application development on the chain is greatly reduced. It means that new developers who even does not know blockchain well can build cApp just by Javascript and publish it nearly free.
```

Comment:
Can you explain what development with Anchor instead of, say, Substrate or ink looks like and how it is easier?

Reply:
> Can you explain what development with Anchor instead of, say, Substrate or ink looks like and how it is easier?

The reason is that developer who just has the skill of pure Javascript can join the blockchain development.
The follow is the structure of EasyPolka, after the green part developed well, only the two red parts left to care about.

![EasyPolka_bluemap.png](http://android.im/anchor/EasyPolka_bluemap.png)

1. cApp part, it is pure JS coding. I think after some basic work, current JS applications can be converted to it.
2. vService part, it is based on Node.js right now. Some of the services such as the cache for the on-chain data can be shared. Only some new function need to develop.

About ink or smart contract, I am not working on them actually, so just my view. 
They are used for Financial Application very well. But for the normal applications, coins is not always the main thing to think about. That's Anchor pallet benefit, care about trusted data instead of coins.

Reply:
Sorry, missing some details.
I have tried to work on smart contract , but failed. I can read the code , but can not make it work.
Will try again, I think I know substrate much better than the point I tried it, :-)